backend/api/resources/accounts/resources.py

```
@@ -14,46 +13,17 @@ class AccountsResource(BaseResource):
     def post(self):
         """User Registration (new account, user and store)"""
 
-        resource_parser = reqparse.RequestParser(trim=True, bundle_errors=True)
+        if request.is_json:
+            data = request.get_json()
+        else:
+            data = None
```

This will not work properly.
data, with this implementation has 2 types, None or dict.
I would fail the request with a HTTP 415 Unsupported Media Type possibly? We only care for Json.

---

It fails when you dont sent header application/json